README.md

```diff
@@ -62,6 +63,47 @@ To install, do:
 $ pip install snakepit
 ```
 
+## Use PyRun as Python distribution to build a spec-file
+
+You do not need any make-opt-writable rpm or svn2rpm. Simply build your spec-
+file in userspace and reduce the rpm size more than by half.
+
+### Special optional configuration for PyRun
+The PyRun binary version 2.1.1 we download was compiled against OpenSSL libraries
+which are different in minor version and naming from the one on server we wan't 
+to install the RPM on
+ 
+The ```libraries``` argument was introduced, so the spec-file can do some
+linkage magic for you.
+It is a dict with {'TARGET': 'LINK_NAME'}
+
+Example:
+```
+libraries: {'/usr/lib64/libssl.so':'libssl.so.1.0.0', '/usr/lib64/libcrypto.so':'libcrypto.so.1.0.0'}
```

Comment:
Why not simply download the PyRun Source and compile it against the build OS as part of the rpmbuild? That way you don't need this library linking at all and you can make sure that the PyRun environment actually fits the target OS.